src/style.css

```diff
@@ -1,3 +1,113 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  color: #060b35;
+  line-height: 14px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+}
+
+.product {
+  margin: 0 auto;
+  height: 134px;
+  width: 160px;
+}
+
+.card__name {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  margin: 40px 0 0;
+}
+
+.card__code {
+  font-size: 10px;
+  color: #616070;
+  margin: 4px 0 0;
+}
+
+.price,
+.reviews {
+  display: flex;
+  justify-content: space-between;
+}
+
+.reviews {
+  align-items: center;
+  margin-top: 16px;
+  font-size: 10px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars .stars__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.reviews__text {
+  align-items: center;
+  font-size: 10px;
+}
+
+.price {
+  margin-top: 24px;
+  margin-bottom: 16px;
+  line-height: 18px;
+}
+
+.price__text {
+  font-size: 12px;
+  color: #616070;
+}
+
+.price__amount {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.buy {
+  display: block;
+  text-decoration: none;
+  text-align: center;
+  background-color: #00acdc;
+  color: #fff;
+  border-radius: 5px;
+  width: 166px;
+  height: 40px;
+  line-height: 40px;
+  font-size: 14px;
+  font-weight: 700;
+  cursor: pointer;
+  transition: 0.3s;
+}
+
+.buy:hover {
+  color: #00acdc;
```

Comment:
Move all repeatable colors to the variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)